src/main/java/frc/robot/subsystems/lights/Lights.java

```diff
@@ -1,7 +1,93 @@
 package frc.robot.subsystems.lights;
 import frc.robot.Constants;
+import com.ctre.phoenix.led.CANdle;
+import com.ctre.phoenix.led.CANdle.LEDStripType;
+import com.ctre.phoenix.led.CANdleConfiguration;
+import com.ctre.phoenix.led.RainbowAnimation;
+import com.ctre.phoenix.led.StrobeAnimation;
 import frc.robot.RobotMap;
+import java.util.TreeMap;
 
 public class Lights {
-    
-}
+  /**
+   * Tree map of light code enums and integers that represent the R, G, B values of each light code
+   * enum
+   */
+  private TreeMap<LightCode, Integer[]> lightOptionsMap;
+  
+  private CANdle candle = new CANdle(LightsConstants.CANDLE_CAN_ID);
+  private CANdleConfiguration config = new CANdleConfiguration();
+  private LightCode currentLightStatus = LightCode.OFF;
+ 
+  public enum LightCode {
+    DISABLED,
+    READY_TO_INTAKE, // ORANGE
+    HAS_CORAL, // GREEN
+    SCORE_PREP, // BLUE
+    READY_TO_SCORE, // YELLOW
+    CLIMB_PREP_SHALLOW, // WHITE
+    CLIMB_PREP_DEEP, // RED
+    READY_TO_CLIMB, // PURPLE
```

Comment:
add an enum value for CLIMBING
and an enum value called PARTY_MODE (this should be the rainbow one)